validation-test/compiler_crashers/28263-swift-typechecker-validatedecl.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: not --crash %target-swift-frontend %s -parse
+// REQUIRES: asserts
+
+// Distributed under the terms of the MIT license
```

Comment:
All code contributed to Swift is under the Apache 2 license.  Is there a reason to include this MIT license comment?